client/global/sass/sections/_navigation.scss

```diff
@@ -17,7 +17,14 @@ $topbar-height:  80px;
 .nav-logo {
   display: inline-block;
   @include large-font;
-  color: #fff;
+  color: rgba(250, 150, 0, 0.9);
+  font-weight: 700;
+}
+
+.nav-otherlogo {
+  display: inline-block;
+  @include largest-font;
+  color: rgba(250, 150, 0, 0.9);
   font-weight: 700;
 }
 
```

Comment:
this could use the features of SASS better.  Could be written as: 

```
.nav-logo {
  //- default things here 
  ...
  &.other {
      //- other-logo specific things here 
      ....
   }
}
``` 

in the compiled CSS, this will translate to ``` .nav-logo.other ```